### DISCUSSION TITLE
How to stay up to date with renovate releases?

### DISCUSSION THREAD

**Discussion by bavedarnow** [renovatebot/renovate#9037] [Request Help] [answered]
For our setup, we're using a custom docker image that pulls `FROM renovate/renovate:24.74` and have recently enabled renovate to run against itself to update that docker tag, but the PR it creates doesn't contain the release notes. Is there a better way to keep up with the new renovate releases/features? Should the packagerules check github instead of docker hub so it can pull in the change log?

![image](https://user-images.githubusercontent.com/785186/110337340-d149fe80-7ff3-11eb-8371-e23abe749e9c.png)

**Comment by viceice** [comment, marked as answer]
You should include the patch release. then it should work

**Reply by bavedarnow** [reply]
Thanks, that did it
